pkgs/applications/misc/acpic/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "acpic";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    inherit pname;
+    sha256 = "sha256-vQ9VxCNbOmqHIY3e1wq1wNJl5ywfU2tm62gDg3vKvcg=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.pbr
+  ];
+
+  format = "setuptools";
```

Comment:
Ah, down here. Please move it just below  version.

Reply:
done